custom_model_runner/datarobot_drum/drum/drum.py

```diff
@@ -1055,18 +1055,7 @@ def _get_requirements_lines(reqs_file_path):
         ret_docker_image = None
         if os.path.isdir(docker_image_or_directory):
             docker_image_or_directory = os.path.abspath(docker_image_or_directory)
-            # Set image tag to the dirname/dirname of the docker context.
-            # E.g. for two folders:
-            # /home/path1/my_env
-            # /home/path2/my_env
-            # tags will be 'path1/my_env', 'path2/my_env'
-            #
-            # If tag already exists, older image will be untagged.
```

Comment:
nit, do you want to keep this "if tag already exists" related comment? Like maybe move it to this part?
https://github.com/datarobot/datarobot-user-models/blob/5e6e6ac6a8cc9d58835d01b6eeb2942e9143c9bc/custom_model_runner/datarobot_drum/drum/drum.py#L1115-L1116